sys-libs/libndp.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import stdlib
+from stdlib.split.drain_all import drain_all_with_doc
+from stdlib.template import autotools
+from stdlib.manifest import manifest
+from stdlib.template.configure import configure
+
+
+@manifest(
+    name='libndp',
+    category='sys-libs',
+    description='''
+    Library for Neighbor Discovery Protocol.
+    ''',
+    tags=['network', 'ndptool'],
+    maintainer='dorian.trubelle@epitech.eu',
+    licenses=[stdlib.license.License.CUSTOM],
+    upstream_url='http://libndp.org/',
+    kind=stdlib.kind.Kind.EFFECTIVE,
+    versions_data=[
+        {
+            'semver': '1.7.0-1',
+            'fetch': [{
+                    'url': 'http://libndp.org/files/libndp-1.7.tar.gz',
+                    'sha256': '2c480afbffb02792dbae3c13bbfb71d89f735562f2795cca0640ed3428b491b6',
+                },
+            ],
+        },
+    ],
+)
+def build(build):
+    packages = autotools.build(
+        configure=lambda: configure(
+            '--localstatedir=/var',
```

Comment:
This flag is already provided by the configure template.

Reply:
I couldn't find this in the documentation, did I miss it?

Reply:
The documentation says that `configure()` inflates the content of `get_dir_flags()`, and the documentation of `get_dir_flags()` says:

```
Build a list of directory flags used by the configure() template.

The return value is a list of directory flags that indicates where the build software should be located in the filesystem.

For example, it will provide flags like:

--prefix='/usr'
--bindir='/usr/bin'
etc.
```

It is not explicitely stated that `--localstatedir` is appended, but it suggests that a bunch of `*dir` flags are set.

([Source](https://docs.raven-os.org/p/nbuild/new/stdlib.template.configure.html))